### ISSUE TITLE
Most often roundcube webmail endlessly waits while replying emails

### ISSUE THREAD

**Issue by asierrl** [roundcube/roundcubemail#7877]
Most of the times I try to reply an email it won't be sent. Roundcube webmail stacks at the warning "Sending..." and nothing happens. I've been trying to fix it for more than one year, assuming I was looking for some settings I would need to change, with o success till now. I have not either been able to identify when it happens (but it is the most common reaction when replying, only seldom am I allowed to finally send the reply). The same happens if I then try to save the message.
I have three email accounts on my server and it happens with all of them, when using roundcube webmail, but it does not when using BlueMail in my smartphone.
How can I fix the issue, any hints?
Best regards, 
Asier

**Comment by alecpl** [comment]
This can be anything. For example it takes too long to send mail or save it in Sent folder. It might be also some client-side issue. Do you see any errors in browser console when that happens (maybe in Console or Network tab)?

**Comment by asierrl** [comment]
Yes, I tried again with the console open (I open the console, then enter the webmail address, open a message and click on Resend to) and I found:
* A 403 Forbidden 95 ms message in the console tab.
* Same error on the network tab. 
I attach a screenshot of both of them, in case some of the information is useful for you. 
Thanks for helping...
![imagen](https://user-images.githubusercontent.com/8058052/107624353-06fd0100-6c5b-11eb-830e-a0efacf2d74e.png)
